extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -34,11 +34,11 @@
      * The pipeline will only be started once all these topics are present in the Kafka cluster.
      */
     @ConfigItem
-    public Optional<String> topics;
+    public List<String> topics;
 
     @Override
     public String toString() {
         return "KafkaStreamsRuntimeConfig [applicationId=" + applicationId + ", bootstrapServers=" + bootstrapServers
-                + ", applicationServer=" + applicationServer + ", topics=" + topics + "]";
+                + ", applicationServer=" + applicationServer + ", topics=[" + String.join(",", topics) + "]]";
```

Comment:
You could have just used `topics`. The default `toString()` will do what you did automatically.

Reply:
You're right. I will revert this.
Thanks for th review ;)